rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/reports/TabulatorGenerator.xtend

```diff
@@ -218,6 +255,12 @@ class TabulatorGenerator {
 		}
 	}
 	
+	private def boolean isTypeTabulatable(Data type) {
+		val types = rosettaConfiguration.generators.tabulators.types
+		val fqn = String.format("%s.%s", type.model.name, type.name)
```

Comment:
i will push an update to this now ... ;-)

Reply:
I think this is alright for now. In our POC I've created a utility for this, but that's not production code yet...